src/core/thread/child_mask.hpp

```diff
@@ -58,9 +60,20 @@ namespace ot {
  * This class represents a bit-vector of child mask.
  *
  */
-class ChildMask
+class ChildMask : public Equatable<ChildMask>, public Clearable<ChildMask>
+
 {
 public:
+    /**
+     * This method clears all the bytes of ChildMask.
+     *
+     * `using` here to bring the overload into the derived class to
+     * participate in overload resolution.
+     *
+     */
+
+    using Clearable<ChildMask>::Clear;
```

Comment:
Clear() and Clear(uint16_t aChildIndex) share same name which is a little misleading, that's why I didn't use `Clearable<ChildMask>` in ChildMask